src/App.jsx

```diff
@@ -14,57 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const clearSelectedGood = () => setSelectedGood(null);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : `No goods selected`}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => clearSelectedGood()}
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            const isSelectedGood = selectedGood === good;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={isSelectedGood ? 'has-background-success-light' : ''}
+              >
+                <td>
+                  <button
+                    data-cy={!isSelectedGood ? 'AddButton' : 'RemoveButton'}
+                    type="button"
+                    className={`button ${isSelectedGood ? 'is-info' : ''}`}
+                    onClick={() => {
+                      if (isSelectedGood) {
+                        clearSelectedGood();
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+                        return;
+                      }
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+                      setSelectedGood(good);
+                    }}
+                  >
+                    {!isSelectedGood ? '+' : '-'}
```

Comment:
```suggestion
                    {isSelectedGood ? '-' : '+'}
```

It's easier for understanding